### PR TITLE
Lk ss2 maintenance

### PR DESCRIPTION
**Pull request by ekiernan**
### Purpose
Minor formatting updates to the SS2 Single and Multi Sample documentation and a new reference to the SS2 Terra workspace in both documents.

### CODE REVIEW

File: pipelines/smartseq2_multisample/README.md [hunk 1]
```diff
@@ -4,22 +4,24 @@
...
+The Smart-seq2 Multi Sample pipeline is a wrapper around the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is developed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files and optionally generating a Loom file.
```
Comment (pshapiro4broad) [inline pipelines/smartseq2_multisample/README.md:7]: Change `[Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample)` to use a relative path too, unless there's a reason that it shouldn't be changed.

File: pipelines/smartseq2_multisample/README.md [hunk 2]
```diff
@@ -4,22 +4,24 @@
...
 
-Full details about the Smart-seq2 pipeline can be read in the [Smart-seq2 Single Sample workflow documentation](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) in GitHub. 
+Full details about the Smart-seq2 pipeline can be read in the [Smart-seq2 Single Sample workflow](/pipelines/smartseq2_single_sample) documentation in GitHub. 
+
+The Multi Sample Smart-seq2 workflow can also be run in [Terra](https://app.terra.bio), a cloud-based analysis platform. The Terra [Smart-seq2 public workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/HCA%20Smart-seq2%20Multi%20Sample%20Pipeline) contains the Smart-seq2 workflow, workflow configurations, required reference data and other inputs, and example testing data.
```
Comment (pshapiro4broad) [inline pipelines/smartseq2_multisample/README.md:11]: Regarding links into terra, is there a more permanent link that can be used here? If this pipeline is no longer featured at some point, will this link continue to work?

File: pipelines/smartseq2_single_sample/README.md [hunk 3]
```diff
@@ -110,7 +112,7 @@ Overall, the workflow is divided into two parts that are completed after an init
...
+HISAT2 is a fast, cost-efficient alignment tool that can determine the presence of non-transcript sequences and true transcript sequences, taking into account the presence of single-nucleotide polymorphisms ([Kim et al.,2019](https://www.nature.com/articles/s41587-019-0201-4)). The Smart-seq2 Single Sample workflow uses the [HISAT2 task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) to call HISAT2 and perform graph-based alignment of paired- or single-end reads (in the form of FASTQ files) to a reference genome. This task  requires a reference index which can be built following the [HCA's build_indices](https://github.com/HumanCellAtlas/skylab/tree/master/library/accessory_workflows/build_indices) documentation. The outputs of the task include a genome-aligned BAM file, a BAM index, and an alignment log file.
```
Comment (pshapiro4broad) [inline pipelines/smartseq2_single_sample/README.md:115]: This line also contains two full paths into skylab that could be changed to use relative paths.

File: pipelines/smartseq2_single_sample/README.md [hunk 4]
```diff
@@ -126,7 +128,7 @@ The [Picard task](https://github.com/HumanCellAtlas/skylab/blob/master/library/t
 
 ### Part 2: Transcriptome Quantification Tasks
 #### 2.1 Align reads to the transcriptome using HISAT2
-The [HISAT2RSEM task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) uses HISAT2 to align reads to the reference transcriptome. The task requires the hisat2_ref_trans_index file and the sample fastq files as input. The output of this task is a transcriptome-aligned BAM file and an alignment log file.
+The [HISAT2RSEM task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) uses HISAT2 to align reads to the reference transcriptome. The task requires the hisat2_ref_trans_index file and the sample FASTQ files as input. The output of this task is a transcriptome-aligned BAM file and an alignment log file.
```
Comment (pshapiro4broad) [inline pipelines/smartseq2_single_sample/README.md:131]: Another non-relative path. Looking at this document, there are many non-relative paths used. Is one form preferred over the other? To me, relative paths look like they'd be easier to edit and maintain. If we do want to move to non-relative paths, maybe a separate JIRA task is needed to fix up all the cases in our current docs. Alternatively, if full paths are preferred, then we shouldn't be moving some references to relative paths.

Comment (ekiernan) [reply pipelines/smartseq2_single_sample/README.md]: I have been trying to move to relative paths as I'm cleaning up docs. I just did another pass through this one and hopefully got them all!

File: pipelines/smartseq2_single_sample/README.md (hunk 3 above)
Comment (ekiernan) [reply pipelines/smartseq2_single_sample/README.md]: I changed these two to relative paths.

File: pipelines/smartseq2_multisample/README.md (hunk 2 above)
Comment (ekiernan) [reply pipelines/smartseq2_multisample/README.md:11]: This is the link. It's actually not a featured workspace (it's a public workspace), it's just under featured workspace billing account. Unless we change the name or funding, the workspace URL will remain the same.

File: pipelines/smartseq2_multisample/README.md (hunk 1 above)
Comment (ekiernan) [reply pipelines/smartseq2_multisample/README.md]: Changed.

**Pull request merged**